ui/package.json

```diff
@@ -14,14 +14,15 @@
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "cy:open": "cypress open",
-    "cy:run": "cypress run",
     "test:unit": "react-scripts test",
     "test:unit:ci": "CI=true react-scripts test --runInBand",
     "tests:pre-commit": "CI=true react-scripts test --findRelatedTests",
-    "e2e-setup": "BROWSER=none start-server-and-test start 3000",
-    "test:e2e": "npm run e2e-setup cy:open",
-    "test:e2e:ci": "npm run e2e-setup cy:run",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+	"start:api": "cd .. && SPRING_PROFILES_ACTIVE=local ./gradlew bootRun withH2",
+	"cy:setup": "BROWSER=none start-server-and-test start:api 8080",
```

Comment:
We might want to name this something like `cy:start-server` since it doesn't build the UI to bundle it with the API and could be misleading with a name like "setup"